api-testing/multicloud-api/build.gradle

```diff
@@ -0,0 +1,24 @@
+plugins {
+    id 'java'
+}
+
```

Comment:
What is the sourceCompatibility of this project?
Will it for work all versions of JAVA?

Reply:
Yes.

Reply:
Please mention this whether it will work for JAVA 5, 6, 7, 8, 9, 10 etc.. seamlessly
Or Fix the source type for this project to work?

Reply:
I have pushed the changes as per your review comments. Please review.

Reply:
What is the exact change you pushed for this comment?

You can simply define.
sourceCompatibility = 1.8

Reply:
I have added  sourceCompatibility. Please review.